code/modules/clothing/shoes/clown.dm

```diff
@@ -5,14 +5,15 @@
 	inhand_icon_state = "clown_shoes"
 	slowdown = SHOES_SLOWDOWN+1
 	var/enabled_waddle = TRUE
+	var/list/squeak_sound = list('sound/effects/clownstep1.ogg'=1,'sound/effects/clownstep2.ogg'=1) //allows clownshoe subtypes to have different sounds
```

Comment:
```suggestion
    ///List of sounds we can choose from on movement of the clown.
	var/list/squeak_sound = list('sound/effects/clownstep1.ogg'=1,'sound/effects/clownstep2.ogg'=1) 
```

Would you mind [dmdoccing](https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md) this? You just have to write the comment, then the variable as such. This allows the language server and codedocs.tgstation13.org to be able to parse it. I wrote an example one for you, but it's likely that this suggestion will probably break your code somehow. I would do a similar edit on your local client (moving the comment up a line, being a bit more general in the statement, and prefixing the comment with `///`).

Thank you!

Reply:
i think i did it but let me know if that's wrong

Reply:
Looks much better to my eyes now, thank you again!